local/languages/finna/en-gb.ini

```diff
@@ -908,6 +908,12 @@ R2_register_read_more_html = "<a href="%%url%%">Read more about the service</a>.
 R2_register_weak_loginmethod = "Finna Reading Room requires strong authentication. Log out and verify your identify using the Suomi.fi e-Identification service."
 R2_rems_connect_error = "An error occurred. Finna Reading Room component not responding. We apologise for any inconvenience caused."
 R2_rems_session_expired = "Your session was terminated for security reasons. You can log in to the Reading Room again."
+reserve_material = "Reserving material"
```

Comment:
"Reservation of material" ?

Reply:
Tarkoitatko itse käännöstä? (koska ne otin tiketissä olevasta käännös-dokumentista)

Reply:
Okei, olkoon sitten.